### PR TITLE
feat: Update homepage

### PR DESCRIPTION
**Pull request by Manav2304**

### CODE REVIEW

File: src/Components/Footer/constant.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+export const templeTimes =
+  "Samadhi Arati: 04:10AM<br>Mangala Arati: 04:30AM<br>Shringar Arati: 07:15AM<br>Guru Puja: 07:25AM<br>Bhagavatam: 08:00AM<br>Pushpa Arati: 08:30AM<br>Raj Bhoga Arati: 12:00PM<br>Temple Closes: 12:45PM<br>Utthapan Arati: 04:00PM<br>Sandhaya Arati: 06:30PM<br>Shayan Arati: 08:00PM<br>Temple Closes: 08:15PM";
+
+export const socialmediaIcon = [
```
Comment (vaidehi27) [inline src/Components/Footer/constant.ts]:
```suggestion
export const socialMediaIcons = [
```

File: src/Components/Footer/constant.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    href: "#",
+    icon: "linkedin",
+  },
+  {
+    name: "Telegram",
+    href: "https://t.me/iskconvvn",
+    icon: "telegram",
+  },
+];
+export const contactInfo = [
```
Comment (vaidehi27) [inline src/Components/Footer/constant.ts]:
```suggestion
export const contactInfos = [
```

File: src/Components/Footer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+      <section className="d-flex justify-content-center justify-content-lg-between p-4 border-bottom">
+        <MDBContainer className="text-center text-md-start mt-5">
+          <MDBRow className="mt-3">
+            <MDBCol md="3" lg="4" xl="3" className="mx-auto mb-4">
+              <h6 className="text-uppercase fw-bold mb-4">
+                <MDBIcon color="secondary" icon="gem" className="me-3" />
+                Timings
+              </h6>
+              <div dangerouslySetInnerHTML={{ __html: templeTimes }} />
```
Comment (vaidehi27) [inline src/Components/Footer/index.tsx]: What is this? what is it for?

File: src/Components/Footer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                ))}
+              </ContactStyle>
+            </MDBCol>
+          </MDBRow>
+        </MDBContainer>
+      </section>
+
+      <div
+        className="text-center p-4"
+        style={{ backgroundColor: "rgba(0, 0, 0, 0.05)" }}
```
Comment (vaidehi27) [inline src/Components/Footer/index.tsx]: no inline style.

File: src/Components/Footer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+              <h6 className="text-uppercase fw-bold mb-4">
+                <MDBIcon color="secondary" icon="gem" className="me-3" />
+                Timings
+              </h6>
+              <div dangerouslySetInnerHTML={{ __html: templeTimes }} />
+            </MDBCol>
+
+            <FooterLogoStyle>
+              <MDBRow>
+                <img src={footerlogo} />
```
Comment (vaidehi27) [inline src/Components/Footer/index.tsx]: alt attribute is missing. Every image should have an alt attribute.

File: src/Components/Footer/constant.ts [hunk 6]
```diff
@@ -0,0 +1,65 @@
+export const templeTimes = [
```
Comment (vaidehi27) [inline src/Components/Footer/constant.ts]:
```suggestion
export const schedules = [
```

File: src/Components/Footer/index.tsx [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+          <MDBRow className="mt-3">
+            <MDBCol md="3" lg="4" xl="3" className="mx-auto mb-4">
+              <h6 className="text-uppercase fw-bold mb-4">
+                <MDBIcon color="secondary" icon="gem" className="me-3" />
+                Timings
+              </h6>
+              <div>
+                {templeTimes.map((time, index) => (
+                  <p key={index}>
+                    {time.name}: {time.time}
```
Comment (vaidehi27) [inline src/Components/Footer/index.tsx]:
```suggestion
                {schedules.map(schedule => (
                  <p key={schedule.name}>
                    {schedule.name}: {schedule.time}
```

Avoid using index as key wherever possible

File: src/Components/Dropdown/index.tsx [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+          <Dropdown.Item href="/Pages/Servicespage/DevoteeKitchen">
+            Devotee Kitchen
+          </Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/KirtanAtIskcon">
+            Kirtan At Iskcon
+          </Dropdown.Item>
+          <Dropdown.Item href="/Pages/Servicespage/BookDistribution">
+            Book Distribution
+          </Dropdown.Item>
+        </Dropdown.Menu>
```
Comment (vaidehi27) [inline src/Components/Dropdown/index.tsx]: I don't see routes set for all these paths in App.tsx. Why is it here? Also, I said the path should not start with `Pages` and should not have capital letters

Comment (vaidehi27) [reply src/Components/Dropdown/index.tsx]: Also you can use map function here

File: src/Components/Footer/index.tsx [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+                  ))}
+                </ContactStyle>
+              </MDBCol>
+            </MDBRow>
+          </MDBContainer>
+        </section>
+
+        <div
+          className="text-center p-4"
+          style={{ backgroundColor: "rgba(0, 0, 0, 0.05)" }}
```
Comment (vaidehi27) [inline src/Components/Footer/index.tsx]: Remove inline style

File: src/index.css [hunk 10]
```diff
@@ -1,13 +1,15 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url("./assets/bgimage.png");
```
Comment (vaidehi27) [inline src/index.css]:
```suggestion
  background-image: url("./assets/images/bgimage.png");
```

File: src/Components/Navbar/index.tsx [hunk 11]
```diff
@@ -1,30 +1,33 @@
...
+      <nav className="navbar  ">
+        <div className="container">
+          <DropDown />
+          <StyledLink to="#">GO Seva</StyledLink>
+          <StyledLink to="#">Festival</StyledLink>
+          <StyledLink to="#">Services</StyledLink>
+          <StyledLink to="/">
+            <img src={logo} alt="Logo" />
+          </StyledLink>
+          <StyledLink to="#">Conatact</StyledLink>
```
Comment (vaidehi27) [inline src/Components/Navbar/index.tsx]:
```suggestion
          <StyledLink to="#">Contact</StyledLink>
```

File: src/Components/Navbar/index.tsx [hunk 12]
```diff
@@ -1,30 +1,33 @@
...
-            <Link className="nav-link" to="Pages/Donation">
-              Donation
-            </Link>
-            <Link className="nav-link" to="#">
-              GO SEVA
-            </Link>
+      <nav className="navbar  ">
+        <div className="container">
+          <DropDown />
+          <StyledLink to="#">GO Seva</StyledLink>
```
Comment (vaidehi27) [inline src/Components/Navbar/index.tsx]:
```suggestion
          <StyledLink to="#">Go Seva</StyledLink>
```

File: src/App.tsx [hunk 13]
```diff
@@ -1,24 +1,34 @@
...
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-
-import NavigationBar from "./Components/Navbar";
-import MyCarousel from "./Pages/Home";
-import AboutUs from "./Pages/About";
-import Home from "./Pages/Home";
-import DonationPage from "./Pages/Donation";
+import { Routes, Route, Link, BrowserRouter } from "react-router-dom";
+import { NavigationBar } from "./Components/Navbar/index";
+import HomePage from "./Pages/Home";
```
Comment (vaidehi27) [inline src/App.tsx]:
```suggestion
import { HomePage } from "./Pages/Home";
```

To be consistent with other pages, make this change

File: src/Pages/History/index.tsx [hunk 14]
```diff
@@ -0,0 +1,12 @@
+import TextSection from "../dropdowninfo/TesxtSection";
```
Comment (vaidehi27) [inline src/Pages/History/index.tsx]:
```suggestion
import TextSection from "../dropdowninfo/TextSection";
```

File: src/Pages/Home/index.tsx [hunk 15]
```diff
@@ -1,110 +1,51 @@
...
-      </Background>
+    <div>
+      <ReactCarousel />
+      <HeadingStyle>
+        <h1>ABOUT US</h1>
+        <Image src={TitlebaseUrl} />
+        <ImageStyle>
+          <img src={about} className="rounded" />
+        </ImageStyle>
+        <TextStyle> {mapAboutUs}</TextStyle>
```
Comment (vaidehi27) [inline src/Pages/Home/index.tsx]:
```suggestion
        <TextStyle> {aboutUs.map(paragraph => <p>{paragraph}</p>)}</TextStyle>
```

Apply this everywhere and remove code line 8 to 14

File: src/Pages/Home/index.tsx [hunk 16]
```diff
@@ -1,110 +1,63 @@
 import React from "react";
-import { StyledCarousel } from "./style";
-import { Background } from "./style";
+import { ImageStyle, TextStyle, HeadingStyle, Image } from "./style";
+import { aboutUs, history, goals, philosophy, founder } from "./constant";
+import TitlebaseUrl from "../../assets/images/Titel Base 16bit.png";
+import about from "../../assets/images/About.jpg";
+import { ReactCarousel } from "./Reactcarousal";
```
Comment (vaidehi27) [inline src/Pages/Home/index.tsx]:
```suggestion
import { ReactCarousel } from "./ReactCarousel";
```

File: src/Pages/Home/index.tsx [hunk 17]
```diff
@@ -1,110 +1,63 @@
 import React from "react";
-import { StyledCarousel } from "./style";
-import { Background } from "./style";
+import { ImageStyle, TextStyle, HeadingStyle, Image } from "./style";
+import { aboutUs, history, goals, philosophy, founder } from "./constant";
+import TitlebaseUrl from "../../assets/images/Titel Base 16bit.png";
+import about from "../../assets/images/About.jpg";
+import { ReactCarousel } from "./ReactCarousal";
```
Comment (vaidehi27) [inline src/Pages/Home/index.tsx]:
```suggestion
import { ReactCarousel } from "./DsCarousel";
```

File: src/Pages/Home/index.tsx [hunk 18]
```diff
@@ -1,110 +1,63 @@
...
+      <HeadingStyle>
+        <h1>ABOUT US</h1>
+        <Image src={TitlebaseUrl} />
+        <ImageStyle>
+          <img src={about} className="rounded" />
+        </ImageStyle>
+        <TextStyle>
+          {aboutUs.map((paragraph) => (
+            <p key={paragraph}>{paragraph}</p>
+          ))}
```
Comment (vaidehi27) [inline src/Pages/Home/index.tsx]:
```suggestion
          {aboutUs.map(paragraph => <p key={paragraph}>{paragraph}</p>)}
```

File: src/Pages/Home/Reactcarousal.tsx [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+
+export const ReactCarousel = () => {
+  return (
+    <Reviews>
+      <Carousel>
+        {carouselImg.map((carouselItem) => (
+          <Carousel.Item key={carouselItem.id}>
+            <ImageCarousel
+              src={carouselItem.image}
+              alt={`Review ${carouselItem.id}`}
```
Comment (vaidehi27) [inline src/Pages/Home/Reactcarousal.tsx]:
```suggestion
              alt={`Carousel Image ${carouselItem.id}`}
```

File: src/Pages/Home/index.tsx (hunk 15 above)
Comment (Manav2304) [reply src/Pages/Home/index.tsx]: Prettier automatically adds brackets to the  paragraph in the map function

File: src/Pages/dropdowninfo/index.tsx [hunk 20]
```diff
@@ -0,0 +1,31 @@
...
+}) => {
+  const mappedContent = content.map((paragraph) => (
+    <p key={paragraph}>{paragraph}</p>
+  ));
+
+  return (
+    <div>
+      <HeadingStyle>
+        <h2>{title}</h2>
+        <Image src={TitleBase} />
```
Comment (vaidehi27) [inline src/Pages/dropdowninfo/index.tsx]: missing alt tag

File: src/Pages/Home/constant.ts [hunk 21]
```diff
@@ -0,0 +1,130 @@
...
+  {
+    id: 2,
+    image: history2Image,
+  },
+  {
+    id: 3,
+    image: history3Image,
+  },
+];
+export const titleTile = [
```
Comment (vaidehi27) [inline src/Pages/Home/constant.ts]:
```suggestion
export const titleTiles = [
```

File: src/Pages/Home/constant.ts [hunk 22]
```diff
@@ -0,0 +1,130 @@
...
+  "Our mission is to spread the teachings of Lord Krishna and the Vaishnava tradition to all those who seek spiritual knowledge and enlightenment. We believe that the path of devotion to Lord Krishna is the key to achieving inner peace and happiness in life.",
+  "We offer a wide range of religious and spiritual activities for visitors to participate in, including kirtans, bhajans, and discourses on the teachings of Lord Krishna. Our programs are designed to help people connect with their inner selves and develop a deeper understanding of the spiritual world.",
+  "Our temple is located in the town of Vallabh Vidya Nagar in the Anand district of the Indian state of Gujarat.",
+];
+const history = [
```
Comment (vaidehi27) [inline src/Pages/Home/constant.ts]:
```suggestion
const histories = [
```

File: src/Pages/Home/index.tsx [hunk 23]
```diff
@@ -1,110 +1,41 @@
...
-                  className="carousel-control-next-icon"
-                  aria-hidden="true"
-                ></span>
-                <span className="visually-hidden">Next</span>
-              </button>
-            </div>
-          </StyledCarousel>
+    <div>
+      <GlobalCarousel images={carouselImages} />
+      {titleTiles.map((titleTiles) => (
```
Comment (vaidehi27) [inline src/Pages/Home/index.tsx]:
```suggestion
      {titleTiles.map((titleTile) => (
```

File: src/Pages/Home/constant.ts [hunk 24]
```diff
@@ -0,0 +1,130 @@
...
+  },
+  {
+    id: 3,
+    image: history3Image,
+  },
+];
+export const titleTiles = [
+  {
+    title: "ABOUT US",
+    image: about,
```
Comment (vaidehi27) [inline src/Pages/Home/constant.ts]:
```suggestion
    images: [about],
```

File: src/Pages/Home/index.tsx [hunk 25]
```diff
@@ -1,110 +1,41 @@
...
+        <div>
+          <HeadingStyle>
+            <h1>{titleTiles.title}</h1>
+            <ImageTitle src={titleBaseUrl} alt="Title Image" />
+          </HeadingStyle>
+          {titleTiles.images && <GlobalCarousel images={titleTiles.images} />}
```
Comment (vaidehi27) [inline src/Pages/Home/index.tsx]:
```suggestion
          {titleTiles.images.length > 1 && <GlobalCarousel images={titleTiles.images} />}
```

**Review by vaidehi27 — APPROVED** [review]

**Review by vaidehi27 — APPROVED** [review]

**Pull request merged**